src/jit/unwind.h

```diff
@@ -71,7 +71,8 @@ class UnwindBase
     UnwindBase() { }
     ~UnwindBase() { }
 
-#ifdef DEBUG
+// TODO: How do we get the ability to access uwiComp without error on Clang?
+#if defined(DEBUG) && !defined(__GNUC__)
```

Comment:
I have currently just disabled, any suggestion for how it would be possible to get uwiComp as it appears to be a circular include. Or am I better just disabling in this case as it appears to be debug specific anyway.